spec/models/builder_spec.rb

```diff
@@ -173,37 +184,16 @@ def file_contains(path, fragment)
       #   'spec.add_dependency "rails-assets-desandro--get-size", ">= 1.1.4", "< 2.0"'
     end
 
-    # This is strange example because colorbox has only
-    # colorbox.css files that are examples (about 5 of them)
-    # Rails Assets algorithm select first example colorbox.css
-    # because it is named the same way as the gem
-    #
-    # I'm not sure it is bug of feature, so I'm leaving it :-)
-    component "colorbox", "1.5.5" do
-      gem_file "app/assets/stylesheets/colorbox.scss"
-      gem_file "app/assets/stylesheets/colorbox/colorbox.scss"
-      file_contains "app/assets/stylesheets/colorbox.scss",
-        "@import 'colorbox/colorbox.scss';"
-    end
-
-    # main is hash
```

Comment:
Main cannot be a hash in this version of Bower. They removed that function without bumping the major version, disregarding semver.